basicwidget/text.go

```diff
@@ -314,6 +312,7 @@ func (t *Text) HandleInput(context *guigui.Context) guigui.HandleInputResult {
 		return guigui.HandleInputResult{}
 	}
 
+	t.clickTickCounter++
```

Comment:
Hmm, there is no guarantee that HandleInput is called once per tick so far. Let me think